bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -394,10 +396,14 @@ where
 
 	// Convert ERC20 token address to a location that can be understood by Assets Hub.
 	fn convert_token_address(network: NetworkId, token: H160) -> Location {
-		Location::new(
-			2,
-			[GlobalConsensus(network), AccountKey20 { network: None, key: token.into() }],
-		)
+		if token == H160([0; 20]) {
```

Comment:
Add a comment here explaining that a token address of `0x00...` is equivalent to native ether.

Reply:
Address in [d7b33dc](https://github.com/paritytech/polkadot-sdk/pull/6855/commits/d7b33dc8e80776abc8b34fd5fc64f478a05d966c).